src/private/meeting.ts

```diff
@@ -2,7 +2,7 @@ import { ensureInitialized, sendMessageRequestToParent } from '../internal/inter
 import { GlobalVars } from '../internal/globalVars';
 import { SdkError } from '../public/interfaces';
 
-export namespace meetingAudio {
+export namespace meeting {
```

Comment:
So we have a few other namespaces like this which use the plural version - e.g. `conversations` and `settings`. Do you think it would make sense to make this plural as well - i.e. `meetings`?

Reply:
I saw a few namespaces like "media" and "meetingRoom" which are singular. I think it makes more sense for it to be singular.